modules/es7.promise.finally.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// https://github.com/tc39/proposal-promise-finally
+var $export = require('./_export');
+
+$export($export.P, 'Promise', {
+  'finally': function _finally(onFinally){
+    var handler = typeof onFinally === 'function' ? onFinally : function(){};
+    var C = Promise;
```

Comment:
That's why my second open item is asking the right way in core-js to get the species constructor. I think I've found it now (not checked in yet) but I'm still getting `TypeError` with `delay` (see image in initial comment).